rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildConfigurationEndpoint.java

```diff
@@ -152,8 +152,8 @@ public Response getAll(@ApiParam(value = PAGE_INDEX_DESCRIPTION) @QueryParam(PAG
 
     @ApiOperation(value = "Creates a new Build Configuration")
     @ApiResponses(value = {
-            @ApiResponse(code = INVLID_CODE, message = INVALID_DESCRIPTION, response = BuildConfigurationSingleton.class),
             @ApiResponse(code = SUCCESS_CODE, message = SUCCESS_DESCRIPTION, response = BuildConfigurationSingleton.class),
+            @ApiResponse(code = INVLID_CODE, message = INVALID_DESCRIPTION, response = ErrorResponseRest.class),
```

Comment:
Maybe separate then :D
27 kwi 2016 17:03 "Emmett Underhill" <notifications@github.com> napisał(a):

> In
> rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildConfigurationEndpoint.java
> <https://github.com/project-ncl/pnc/pull/957#discussion_r61274939>:
>
> >              @ApiResponse(code = SUCCESS_CODE, message = SUCCESS_DESCRIPTION, response = BuildConfigurationSingleton.class),
> > +            @ApiResponse(code = INVLID_CODE, message = INVALID_DESCRIPTION, response = ErrorResponseRest.class),
>
> Yeah the replace is 111 lines, should I make a new PR for this or just
> commit it here?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/project-ncl/pnc/pull/957/files/60891117459b7c90c1d663f7eadebba324b1e580#r61274939>
>